javascript/elements/index.js

```diff
@@ -34,9 +36,30 @@ const defineElements = e => {
   }
 }
 
+const cleanUp = e => {
+  Object.entries(sessionStorage).forEach(([key, payload]) => {
```

Comment:
If you namespace the sessionStorage this would become something like

```javascript
  const inNamespace = ([key, _payload]) => key.startsWith("futurism-")
  Object.entries(sessionStorage)
             .filter(inNamespace)
             .forEach(([key, payload]) => {
               ...
```

Reply:
thanks for this! that should certainly clear things up!